packages/maskbook/src/plugins/ITO/SNSAdaptor/hooks/useSpaceStationClaimable.ts

```diff
@@ -0,0 +1,12 @@
+import { getClaimableTokenCount } from '../../Worker/apis/spaceStationUUPS'
+import { useAsyncRetry } from 'react-use'
+
+export function useSpaceStationClaimable(address: string) {
+    return useAsyncRetry(async () => {
+        const data = await getClaimableTokenCount(address)
```

Comment:
Are you sure calling those APIs without using RPC?

Reply:
Yes, it's one graphql api, just use `fetch`.